checks/networking/dcos_net_errors.go

```diff
@@ -0,0 +1,76 @@
+package errors
+
+import (
+	"bufio"
+	"fmt"
+	"regexp"
+	"github.com/adyatlov/bun"
+)
+// http://erlang.org/doc/apps/kernel/logger_chapter.html#log_level
+var errorsRegex = "\\[error\\]|\\[critical\\]|\\[alert\\]|\\[emergency\\]"
+
+func init() {
+	builder := bun.CheckBuilder{
+		Name: "net-errors-checker",
+		Description: "Identify errors on dcos-net logs",
+ 		CollectFromMasters:      collect,
+		CollectFromAgents:       collect,
+		CollectFromPublicAgents: collect,
+		Aggregate:               aggregate,
+	}
+	check := builder.Build()
+	bun.RegisterCheck(check)
+}
+
+func collect(host bun.Host) (ok bool, details interface{}, err error) {
+	errorMatcher, _ := regexp.Compile(errorsRegex)
+  keys := 0
+
+	file, err1 := host.OpenFile("net")
+
+	if err1 != nil {
+		ok = false
+		errMsg := fmt.Sprintf("Cant open file %s", err1)
+		fmt.Println(errMsg)
+		return
+	}
+
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+  for scanner.Scan() {
+		line := scanner.Text()
+		if errorMatcher.MatchString(line) {
+			keys++
+		}
+  }
```

Comment:
ditto

Reply:
I have installed some tooling to help me with this, there was mixed tabs and spaces. It looked good on emacs but it was wrong. Sorry for that.